### DISCUSSION TITLE
Need help with netbox admin settings.

### DISCUSSION THREAD

**Discussion by herbatka** [netbox-community/netbox#15881] [Help Wanted!] [answered]
Hey, I've recently upgraded from v3.7.1 to 3.7.6 and everything seemed fine.
Today I noticed that I am missing a lot of settings from the Admin panel on the web interface.
Any ideas what went wrong?
![image](https://github.com/netbox-community/netbox/assets/25375878/2992f2e8-8ac3-4786-9714-2a8615a1fa8d)

**Comment by markkuleinio** [comment, marked as answer]
Version 3.6.0 release notes: https://docs.netbox.dev/en/stable/release-notes/version-3.6/#new-features

**Reply by herbatka** [reply]
Oops, I didn't even think of looking there. Thanks!
